impeller/renderer/backend/gles/proc_table_gles.h

```diff
@@ -72,21 +73,13 @@ template <class... Type>
   return stream.str();
 }
 
-template <class T>
-struct GLProc {
-  using GLFunctionType = T;
-
+struct GLProcBase {
```

Comment:
Why did this get split out?

Reply:
The map that enable "lookup proc by name" needs an un-templated field.